src/Halogen/Svg/Attributes.purs

```diff
@@ -301,4 +302,4 @@ y2 = attr (AttrName "y2") <<< show
 xlinkHref :: forall r i. String -> IProp (xlinkHref :: String | r) i
 -- xlinkHref = attr (AttrName "xlink:href")
 -- xlinkHref = attrNS (Namespace "xlink") (AttrName "href")
```

Comment:
Could you qualify `NameSpace` here, too?

Reply:
Do you mean update the commented out code so that it is still valid if someone uncomments it? If so, fixed with 386037221b63de3181e590c5d77bc339e2fb507c. We probably shouldn't have commented out code, or too many TODO labels. `xlinkHref` is animation related and hopeful we can do something with it in when we fix #21. Also note that `xlinkHref` is depreciated in the SVG standard (https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href).